code/src/main/java/frc/robot/common/IDrivetrainSubsystem.java

```diff
@@ -17,53 +17,53 @@
  */
 public interface IDrivetrainSubsystem extends Subsystem {
```

Comment:
The changes to this file look like all formatting changes that add one extra space, can we revert this?